### DISCUSSION TITLE
Is it possible to import from modules at runtime?

### DISCUSSION THREAD

**Discussion by B-Esmaili** [vlang/v#16717] [Questions and Answers] [answered]
I ran through the documentation but couldn't find something relevant. Is it possible to import functions / types from modules at runtime? If no , do you have any plan to support this feature at the future?

**Comment by larpon** [comment, marked as answer]
It is via dynamic library loading. We have an example here: https://github.com/vlang/v/blob/master/examples/dynamic_library_loading

(See the use.v file)

**Reply by B-Esmaili** [reply]
Fantastic! Thanks for being helpful.

**Reply by larpon** [reply]
Happy to help
